resources/assets/components/SignupCard/signup-card.scss

```diff
@@ -0,0 +1,38 @@
+@import '~@dosomething/forge/scss/toolkit';
+
+$tablet: "(min-width: 760px)" !default;
+$base-spacing: 24px !default;
+$light-gray: #ddd !default;
```

Comment:
Hmm, these should all exist in [Forge's variables](https://github.com/DoSomething/forge/blob/3828463212a96e97b154b0afa007da30b869c5ab/scss/_utilities/_variables.scss) that get imported from the toolkit.